lib/system/selinux/bootstrap.go

```diff
@@ -0,0 +1,525 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package selinux
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"text/template"
+
+	"github.com/gravitational/gravity/lib/defaults"
+	liblog "github.com/gravitational/gravity/lib/log"
+	libschema "github.com/gravitational/gravity/lib/schema"
+	"github.com/gravitational/gravity/lib/system/selinux/internal/policy"
+	"github.com/gravitational/gravity/lib/system/selinux/internal/schema"
+	"github.com/gravitational/gravity/lib/utils"
+
+	"github.com/gravitational/satellite/monitoring"
+	"github.com/gravitational/trace"
+	"github.com/opencontainers/selinux/go-selinux"
+	log "github.com/sirupsen/logrus"
+)
+
+// Bootstrap configures SELinux on the node
+func Bootstrap(config BootstrapConfig) error {
+	if err := config.checkAndSetDefaults(); err != nil {
+		return trace.Wrap(err)
+	}
+	b := newBootstrapper(config)
+	err := utils.WithTempDir(func(dir string) error {
+		return b.installDefaultPolicy(dir)
+	}, "policy")
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	return b.importLocalChanges()
+}
+
+// Unload removes the policy modules and local modifications
+func Unload(config BootstrapConfig) error {
+	if err := config.checkAndSetDefaults(); err != nil {
+		return trace.Wrap(err)
+	}
+	b := newBootstrapper(config)
+	return b.removeLocalChanges()
+	// TODO: remove the policy module.
+	// Note, this is only possible when there're no more gravity-labeled
+	// files/directories.
+}
+
+// IsSystemSupported returns true if the system specified with given ID
+// is supported
+func IsSystemSupported(systemID string) bool {
+	switch systemID {
+	case "centos", "rhel":
+		return true
+	default:
+		return false
+	}
+}
+
+// Patch executes the patch script with the underlying configuration
+func (r PatchConfig) Patch() error {
```

Comment:
So this seems to be dealing specifically with custom vxlan ports, so maybe call this function so? PatchVxlan. And update the godoc respectively.